CondCore/CondDB/src/base64.cc

```diff
@@ -154,161 +135,176 @@ base64_encode_alloc (const char *in, size_t inlen, char **out)
-   : (_) == 'l' ? 37                            \
-   : (_) == 'm' ? 38                            \
-   : (_) == 'n' ? 39                            \
-   : (_) == 'o' ? 40                            \
-   : (_) == 'p' ? 41                            \
-   : (_) == 'q' ? 42                            \
-   : (_) == 'r' ? 43                            \
-   : (_) == 's' ? 44                            \
-   : (_) == 't' ? 45                            \
-   : (_) == 'u' ? 46                            \
-   : (_) == 'v' ? 47                            \
-   : (_) == 'w' ? 48                            \
-   : (_) == 'x' ? 49                            \
-   : (_) == 'y' ? 50                            \
-   : (_) == 'z' ? 51                            \
-   : (_) == '0' ? 52                            \
-   : (_) == '1' ? 53                            \
-   : (_) == '2' ? 54                            \
-   : (_) == '3' ? 55                            \
-   : (_) == '4' ? 56                            \
-   : (_) == '5' ? 57                            \
-   : (_) == '6' ? 58                            \
-   : (_) == '7' ? 59                            \
-   : (_) == '8' ? 60                            \
-   : (_) == '9' ? 61                            \
-   : (_) == '+' ? 62                            \
-   : (_) == '/' ? 63                            \
-   : -1)
+#define B64(_)                                                                                                                                                                                                                                                                                                                                                                                          \
+  ((_) == 'A'                                                                                                                                                                                                                                                                                                                                                                                           \
+       ? 0                                                                                                                                                                                                                                                                                                                                                                                              \
+       : (_) == 'B'                                                                                                                                                                                                                                                                                                                                                                                     \
```

Comment:
@smuzaffar @ggovi this change does not make sense, although it is formally according to the criteria. In trying to play with the file I see it is code coming from FSF formally not to be edited. What do we do? Do we try to modify it anyway before formatting, or do we want to exclude it from the formatting?

Reply:
agree this does not make any sense. I will check if one can ignore a section of code to be formated.

Reply:
@fabiocos , see https://github.com/cms-sw/cmssw/pull/26975/commits/de8fe67092b57c63425e8e97add3b6cf6f83765a
I moved backed the original code and turn off the clang-format for this selected region